### PR TITLE
Now stub page for clients with extension will return with plaintext mime-type.

### PR DESCRIPTION
**Pull request by eliduvid**
This allows better security and (accidentally) better performance.

### CODE REVIEW

File: src/main/java/org/moera/node/global/VirtualPageInterceptor.java [hunk 1]
```diff
@@ -35,16 +35,12 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
...
-                    + "    <head>\n"
-                    + "        <meta name='viewport' content='width=device-width, initial-scale=1, shrink-to-fit=no'>\n"
-                    + "        <title>Moera</title>\n"
-                    + "    </head>\n"
-                    + "    <body></body>\n"
-                    + "</html>\n");
+            out.write("Moera client is loading.\n"
+                    + "If you are seeing this message for some time now, something is not right.\n"
+                    + "If Moera extension settings are opened, try closing them.\n"
+                    + "It's not helping? Contact developers though github: https://github.com/MoeraOrg/moera-issues\n");
```
Comment (smelamud) [inline src/main/java/org/moera/node/global/VirtualPageInterceptor.java:43]: Let's not scary user with long explanations what may go wrong ;) Most of the time the loading will work as expected and the extension storage locking bug we will fix in the near future. So let's display just "Loading Moera client..."

**Review by smelamud — CHANGES_REQUESTED** [review]

File: src/main/java/org/moera/node/global/VirtualPageInterceptor.java (hunk 1 above)
Comment (eliduvid) [reply src/main/java/org/moera/node/global/VirtualPageInterceptor.java:43]: If user has time to read it, i think something did go wrong 😃

Comment (eliduvid) [reply src/main/java/org/moera/node/global/VirtualPageInterceptor.java:43]: I think we should just change it to `If you are seeing this message for a couple of minutes` and change formatting a bit

File: src/main/java/org/moera/node/global/VirtualPageInterceptor.java [hunk 2]
```diff
@@ -35,17 +35,10 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
...
-                    + "    <head>\n"
-                    + "        <meta name='viewport' content='width=device-width, initial-scale=1, shrink-to-fit=no'>\n"
-                    + "        <title>Moera</title>\n"
-                    + "    </head>\n"
-                    + "    <body></body>\n"
-                    + "</html>\n");
-            out.close();
+            response.setContentType("text/plain; charset=utf-8");
+            try (OutputStream outputStream = response.getOutputStream()) {
+                outputStream.write("Moera client is loading...".getBytes(StandardCharsets.UTF_8));
```
Comment (smelamud) [inline src/main/java/org/moera/node/global/VirtualPageInterceptor.java:40]: You can use `OutputStreamWriter` with try-with-resources as well.

File: src/main/java/org/moera/node/global/VirtualPageInterceptor.java [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
 import org.springframework.web.method.HandlerMethod;
 import org.springframework.web.servlet.handler.HandlerInterceptorAdapter;
 
+import javax.inject.Inject;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.charset.StandardCharsets;
+
```
Comment (smelamud) [inline src/main/java/org/moera/node/global/VirtualPageInterceptor.java:14]: Why imports are reordered?

**Review by smelamud — CHANGES_REQUESTED** [review]

File: src/main/java/org/moera/node/global/VirtualPageInterceptor.java (hunk 3 above)
Comment (eliduvid) [reply src/main/java/org/moera/node/global/VirtualPageInterceptor.java:14]: No reason. I'll put them back

File: src/main/java/org/moera/node/global/VirtualPageInterceptor.java (hunk 2 above)
Comment (eliduvid) [reply src/main/java/org/moera/node/global/VirtualPageInterceptor.java:40]: I know, it just seams to me we don't need it.

**Review by smelamud — APPROVED** [review]

**Pull request merged**